ansible_base/lib/serializers/mixins.py

```
@@ -1,11 +1,171 @@
+            contain HTML or template syntax (e.g. Jinja2 templates, custom login HTML).
+            Also skips entire JSONFields when listed here.
+        excluded_json_keys: mapping of JSONField name to a frozenset of sub-keys
+            that should be skipped during validation (e.g. PEM keys, template content).
+            Example: {'inputs': frozenset({'ssh_key_data'})}
+
+    See docs/lib/validation.md for the full contract.
+    """
+
+    name_fields = DEFAULT_NAME_FIELDS
+    excluded_fields = frozenset()
+    excluded_json_keys = {}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)